### ISSUE TITLE
Search bar cursor enhancement

### ISSUE THREAD

**Issue by gabkomarnicki** [moezbhatti/qksms#1073]
### DESCRIPTION
After tapping the search bar and then the back button to close the keyboard, the cursor is still blinking in the search bar. Hitting back again closes the app entirely. This doesn't happen if a user searches for something.

Huawei Nova 2i (Mate 10 Lite), 8.0 Oreo

### STEPS
1. Tap the search bar
2. Tap the back button to close the keyboard. The cursor is still there in the search bar blinking
3. Tap back again. The app itself closes.

### EXPECTED
The cursor in the search bar should go away if the keyboard is no longer displayed.

### OBSERVATIONS
When the user closes the keyboard, the cursor would disappear.

**Comment by realmain** [comment]
I don't think it's actually a bug. Majority of my apps do this.

It'd be more of an enhancement.

**Comment by gkomarnicki** [comment]
In Android Messages and Signal, the cursor disappears after hitting the back button, then a second tap on the back button closes the search bar.

In Maps and Google Photos, the cursor remains blinking after the keyboard has been closed.

I guess this is more of an enhancement than a bug, but this is a very minor detail anyway.

**Comment by moezbhatti** [comment]
Yep, would definitely be nice if the cursor disappeared when the keyboard is removed :)
